monitoring/utils/sample-application-under-test/percona/cr.yaml

```diff
@@ -0,0 +1,438 @@
+apiVersion: pxc.percona.com/v1-6-0
+kind: PerconaXtraDBCluster
+metadata:
+  name: cluster1
+  finalizers:
+    - delete-pxc-pods-in-order
+#    - delete-proxysql-pvc
+#    - delete-pxc-pvc
+#  annotations:
+#    percona.com/issue-vault-token: "true"
+spec:
+  crVersion: 1.6.0
+  secretsName: my-cluster-secrets
+  vaultSecretName: keyring-secret-vault
+  sslSecretName: my-cluster-ssl
+  sslInternalSecretName: my-cluster-ssl-internal
+  logCollectorSecretName: my-log-collector-secrets
+  #  disableHookValidation: true
```

Comment:
I suppose we turned this On  prevent ssl errors. @uditgaurav ?

Reply:
should I switch this to on? On my setup I wasn't getting such error but we can disable it for greater compatibility @ksatchit  cc: @uditgaurav